### ISSUE TITLE
[request] inline option group in dialogs

### ISSUE THREAD

**Issue by yeedle** [quasarframework/quasar#940]
For forms in dialogs, can we get an inline option, like we can do with option groups?

Something like

``` js
form: {
    group {
        type: 'toggle',
        model: ['foo', 'bar'],
        items: { ... },
        inline: true
    }
}
```
That would then render the toggles inline instead of row by row.

**Comment by rstoenescu** [comment]
Will add this. Thanks for reporting.

**Comment by yeedle** [comment]
Awesome, thanks!

**Comment by rstoenescu** [comment]
Available in edge and future v0.14.4
